### PR TITLE
Add game-tests for measuring performance, reduce ship chunk ticket le…

### PR DESCRIPTION
**Pull request by StewStrong**
…vel to improve performance

## What this PR does:
- [ ] Bug fix 
- [ ] Feature
- [X] Code refactor / improvement
- [ ] API addition / improvement
- [ ] Compat with another mod

**Explain what this PR is doing:**
Makes code faster and add a gametest

---

## Modloaders this PR affects:
- [x] Forge
- [X] Fabric

## Where this PR was tested:
- [ ] In-dev singleplayer
- [ ] In-dev dedicated server
- [ ] Out of dev singleplayer
- [ X] GameTest framework

## Breaking Changes
- [ ] Yes (describe)
- [X] No
- Probably not
---

## Additional Notes
Anything else reviewers might need to know:
Performance concerns, edge cases, etc

Its 5am and I still haven't gone to sleep yet
---

## Declaration
By submitting this PR, I affirm:  
- Code/data compiles and loads  
- Tests _(if any)_ pass  
- This PR will not brick worlds

### CODE REVIEW

File: common/src/main/kotlin/org/valkyrienskies/mod/common/world/ChunkManagement.kt [hunk 1]
```diff
@@ -31,9 +34,22 @@ object ChunkManagement {
             val chunkPos = ChunkPos(chunkWatchTask.chunkX, chunkWatchTask.chunkZ)
 
             val level = server.getLevelFromDimensionId(chunkWatchTask.dimensionId)!!
-            level.chunkSource.updateChunkForced(chunkPos, true)
 
-            level.server.executeIf({ level.isTickingChunk(chunkPos) }) {
+            // Use lightweight ticket for shipyard chunks to avoid loading excessive neighbor chunks.
+            // Vanilla's updateChunkForced uses level 31 (entity ticking) which forces a 2-chunk
+            // neighborhood (~25 chunks). Our ticket with radius 1 gives level 32 (ticking), which
+            // only needs a 1-chunk neighborhood (~9 chunks) — still a big reduction.
```
Comment (blockninja124) [inline common/src/main/kotlin/org/valkyrienskies/mod/common/world/ChunkManagement.kt:41]: Em dash in comments :sob:
